src/main/java/org/isf/patconsensus/model/PatientConsensus.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Open Hospital (www.open-hospital.org)
+ * Copyright © 2006-2023 Informatici Senza Frontiere (info@informaticisenzafrontiere.org)
+ *
+ * Open Hospital is a free and open source software for healthcare data management.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * https://www.gnu.org/licenses/gpl-3.0-standalone.html
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.isf.patconsensus.model;
+
+import javax.persistence.AttributeOverride;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import javax.persistence.Transient;
+
+import org.isf.patient.model.Patient;
+import org.isf.utils.db.Auditable;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@Entity
+@Table(name = "OH_PATIENT_CONSENSUS")
+@EntityListeners(AuditingEntityListener.class)
+@AttributeOverride(name = "createdBy", column = @Column(name = "PTC_CREATED_BY"))
+@AttributeOverride(name = "createdDate", column = @Column(name = "PTC_CREATED_DATE"))
+@AttributeOverride(name = "lastModifiedBy", column = @Column(name = "PTC_LAST_MODIFIED_BY"))
+@AttributeOverride(name = "active", column = @Column(name = "PTC_ACTIVE"))
+@AttributeOverride(name = "lastModifiedDate", column = @Column(name = "PTC_LAST_MODIFIED_DATE"))
+public class PatientConsensus extends Auditable<String> {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	@Column(name = "PTC_ID")
+	private int code;
+
+
+	@Column(name="PTC_CONSENSUNS")
+	private boolean consensusFlag;
+
+
+	@Column(name="PTC_ADMINISTRATIVE")
+	private boolean administrativeFlag;
+
+
+	@Column(name="PTC_SERVICE")
+	private boolean serviceFlag;
+
+
+    @OneToOne(cascade = CascadeType.ALL)
```

Comment:
Formatting looks out of line here; and there is lots of white space in the file.